pkg/install/discovery.go

```diff
@@ -96,8 +96,8 @@ func FindMatchingService(c context.Context, client *kates.Client, portNameOrNumb
 	portRef := ""
 	if len(matchingSvcs) > 0 {
 		svcNames := make([]string, 0, len(matchingSvcs))
-		for i, svc := range matchingSvcs {
-			svcNames[i] = svc.Name
+		for _, svc := range matchingSvcs {
+			svcNames = append(svcNames, svc.Name)
```

Comment:
Using `append` when the size is know isn't ideal since it creates a new slice instance for each iteration. The `for` loop is correct. The mistake is on the previous line which should't have a middle zero argument. It should just say:
```
svcNames := make([]string, len(matchingSvcs))
```

Reply:
Thanks for your guidance.